src/main/java/com/hazelcast/web/WebFilterConfig.java

```diff
@@ -37,30 +37,135 @@
  * Contains all configuration parameters for Hazelcast session replication
  */
 public final class WebFilterConfig {
-    private static final ILogger LOGGER = Logger.getLogger(WebFilterConfig.class);
+    /**
+     * <p>Configuration key: <code>client-config-location</code></p>
+     * Location of the client's configuration. It can be specified as a servlet resource, classpath resource or as a URL.
+     * Its default value is null.
+     */
+    public static final String CLIENT_CONFIG_LOCATION = "client-config-location";
+
+    /**
+     * <p>Configuration key: <code>session-ttl-seconds</code></p>
+     * Time-to-live value (in seconds) of the distributed map storing your web session objects.
+     * It can be any integer between 0 and {@link java.lang.Integer#MAX_VALUE}.
+     * Its default value is 1800, which is 30 minutes.
+     */
+    public static final String SESSION_TTL_SECONDS = "session-ttl-seconds";
+
+    /**
+     * <p>Configuration key: <code>use-client</code></p>
+     * Specifies whether you want to connect to an existing cluster as a client.
+     * Its default value is false.
+     */
+    public static final String USE_CLIENT = "use-client";
+
+    /**
+     * <p>Configuration key: <code>instance-name</code></p>
+     * Name of an existing Hazelcast instance, if you want to use it.
+     * Its default value is null.
+     * If you do not have an instance, then you should create one.
+     */
+    public static final String INSTANCE_NAME = "instance-name";
+
+    /**
+     * <p>Configuration key: <code>map-name</code></p>
+     * Name of the distributed map storing your web session objects.
+     */
+    public static final String MAP_NAME = "map-name";
+
+    /**
+     * <p>Configuration key: <code>sticky-session</code></p>
+     * If set to true, all requests of a session are routed to the member where the session is first created.
+     * If set to false, when a session is updated on a member, entry for this session on all members is invalidated.
+     * You have to know how your load balancer is configured before setting this parameter.
```

Comment:
Let's eliminate "you"s
```suggestion
     * If set to false, when a session is updated on a member, the updated entry for this session on all members
     * is invalidated. This option must be used in a compatible way with load balancer behavior.
```